tap_google_sheets/streams.py

```diff
@@ -105,6 +108,14 @@ def get_selected_fields(catalog, stream_name):
             pass
     return selected_fields
 
+def new_format_message(message):
+    """To override the ensure_ascii param, overwitten this function"""
+    return json.dumps(message.asdict(), ensure_ascii=False, use_decimal=True)
```

Comment:
Why do we need to override this function?

Reply:
Because while writing the record we need to set ensure_ascii  as False, else it writes the currency symbols as ascii values